src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -35,7 +37,7 @@
  * <code>`a.getId().equalsIgnoreCase(b.getId())`</code> are true then all must
  * be true.</p>
  */
-public interface CatalogType {
+public interface CatalogType extends TextRepresentable {
```

Comment:
> but it can/should due to its nature.

The key word here is **can**, yes, most of them can be. This is the whole point of the human readable name provided by `getName()`. It at the least provides some semblance of what people should "call" this catalog type *should* it be presented in a `Text` object. The point is that not all `CatalogType`s, even ones provided by plugins, fit the bill for being translatable or even describable as `Text`. Plugins seeing this will just ignore the `Text` object that is provided because it's not configurable, modifiable, impractical, and lastly, just doesn't make sense as a general contract of all `CatalogType`s.